packages/klarna-integration/src/klarnav2/klarnav2-payment-strategy.ts

```diff
@@ -257,4 +262,25 @@ export default class KlarnaV2PaymentStrategy {
             );
         });
     }
+
+    private async klarnaOrderInitialization(methodId: string): Promise<void> {
+        const state = this.paymentIntegrationService.getState();
+        const cartId = state.getCartOrThrow().id;
+        const clientToken = state.getPaymentMethodOrThrow(methodId).clientToken;
```

Comment:
```suggestion
        const { id: cartId } = state.getCartOrThrow();
        const { clientToken } = state.getPaymentMethodOrThrow(methodId);
```
let's keep the same code style